styles.css

```diff
@@ -0,0 +1,42 @@
+@import url('https://fonts.googleapis.com/css?family=Barlow+Condensed|Barlow:600|Rasa:300,500');
+
+body {
+    background-image: linear-gradient(rgba(158, 58, 58, 0.8), rgba(206, 136, 136, 0.7)),url(https://img.freepik.com/free-photo/horizontal-view-fresh-delicious-juice-glass-with-orange-lime-tube-dark-background_140725-140821.jpg);
+    background-size: cover;
+    background-repeat: no-repeat;
+    font-family: 'Barlow', sans-serif;
+    color: white;
+    align-items: center;
+}
+
+#header {
+    margin:auto;
+    margin-bottom: 20px;
+    color: rgb(255, 255, 255);
+    background-color:rgba(103, 26, 26, 0.604);
+    text-align: center;
+    padding: 20px;
+    width: 500px;
+    border-radius: 0.4rem;
+
+}
+
+.question-group{
+    padding: 20px;
```

Comment:
✅ Please consider also keeping in mind _Relative Length_ unit's values in order to maintain the responsiveness of your webpage design. By doing so, you can help ensure that your content is displayed proportionally across a range of devices and screen sizes, thereby enhancing the user experience and improving the accessibility of your website.

### Aditional resources
- [CSS Units](https://www.w3schools.com/css/css_units.asp)